index.html

```diff
@@ -0,0 +1,59 @@
+<!DOCTYPE html>
+<html lang="en">
+  <head>
+    <meta charset="UTF-8" />
+    <meta http-equiv="X-UA-Compatible" content="IE=edge" />
+    <meta name="viewport" content="width=device-width, initial-scale=1.0" />
+    <link rel="stylesheet" href="style.css" />
+    <title>Portfolio</title>
+  </head>
+  <body>
+      <div class="main-div">
+          <section class="header-section">
+              <header class="toolbar">
+                  <a href="#">My Logo</a>
+                  <nav>
+                    <img src="image/toggle.png" alt="Toggle" />
+                  </nav>
+              </header>
+          </section>
+          <section class="headline">
+              <h1 class="heading">I am Daisy Glad to see you!</h1>
+              <p class="profile">
+                I’m a software developer! I can help you build a product , feature or
+                website Look through some of my work and experience! If you like what
+                you see and have a project you need coded, don’t hestiate to contact me.
+              </p>
+              <p class="info">LET'S CONNECT</p>
+              <div class="media-icon">
+                  <ul>
+                      <li class="icon">
+                          <a href="#">
+                            <img src="images/twitter.png" alt="Twitter" />
+                          </a>
+                      </li>
+                      <li class="icon">
+                          <a href="#">
+                            <img src="images/Linkedin.png" alt="LinkedIn" />
+                          </a>
+                      </li>
+                      <li class="icon">
+                          <a href="#">
+                            <img src="images/m.png" alt="Medium" />
+                          </a>
+                      </li>
+                      <li class="icon">
+                          <a href="#">
+                              <img src="images/github.png" alt="Github" /></a>
+                      </li>
+                      <li class="icon">
+                          <a href="#">
+                            <img src="images/fingers.png" alt="Finger" />
+                          </a>
+                      </li>
+                  </ul>
```

Comment:
#### OPTIONAL 

Since the last reviewer missed this requirement, I will mark this change as a suggestion 😁

The images on your links are also having issues and are not loading properly 😥 

![image](https://user-images.githubusercontent.com/15807343/148628666-0dbf311b-067f-4661-a89a-21b93b857d36.png)